test/packages/good/manifest.yml

```diff
@@ -4,8 +4,11 @@ title: Good package
 description: This package is good.
 version: 1.0.0
 type: integration
+source:
+  license: "Apache-2.0"
 conditions:
   kibana.version: '^7.9.0'
+  elastic.subscription: 'basic'
```

Comment:
Hm... it's a tough one. It is a condition or requirement to have a `basic` subscription. On the other hand, `conditions` will become a bag with various properties, which isn't great too. Maybe it can be moved to the `licensing` structure I proposed above?

Reply:
We need different fields for license and subscription.

The subscription is a requirement, this is why I thought that putting it as condition could make sense. I am ok with moving it to its own field though.

Reply:
Both options sound good, conditions may also contain regular expressions, for example, `basic || special`. Maybe we can ask other folks about their preferences. @joshdover

Reply:
I was also thinking on the "at least" semantics.

Regarding `basic || special`, do we have any feature now that is included on different subscriptions? This would be for example to have something in `gold` and `enterprise` that is not included in `platinum`. This sounds confusing.

Reply:
> Shouldn't we go ahead and add the other enum values: `basic`, `gold`, `platinum`, `enterprise`?

:+1: added.